src/inputs/BasicInputs/BasicEnumInput/BasicEnumInput.js

```diff
@@ -42,13 +42,29 @@ export const BasicEnumInput = (props) => {
           sx={{ flexGrow: 1 }}
           select
           value={value}
-          onChange={(event) => {
-            return changeEnumField(event.target.value);
-          }}
+          SelectProps={{ renderValue: (value) => <Box>{value}</Box> }} // Prevent extra padding on selected value
+          onChange={(event) => changeEnumField(event.target.value)}
         >
           {enumValues.map((option) => (
-            <MenuItem key={option.key} value={option.key} data-cy={option.key}>
-              {option.value}
+            <MenuItem
+              key={option.key}
+              value={option.key}
+              data-cy={option.key}
+              sx={{ p: '0px' }} // Remove default MenuItem padding to increase the hover area for the items tooltips
+            >
+              <Tooltip
```

Comment:
can we use our `FadingTooltip` component here or it is not suitable for Selects ?

Reply:
I tried using it, but our `FadingTooltip` component adds an intermediary `<div>` that breaks the layout and the tooltip behavior somehow. I didn't take the time to look further into this issue, but we may have to add an optional prop to the `FadingTooltip` to declare whether or not the `<div>` should be added automatically.

Reply:
I confirm that I tried to use FadingTooltip and it doesn' t work correctly